src/backend/joanie/core/fields/schedule.py

```diff
@@ -0,0 +1,21 @@
+"""Utils for the order payment schedule field"""
+
+import datetime
+from json import JSONEncoder
+
+from stockholm import Money
+
+
+class OrderPaymentScheduleEncoder(JSONEncoder):
+    """
+    A JSON encoder for datetime objects.
+    """
+
+    def default(self, o):
+        if isinstance(o, (datetime.date, datetime.datetime)):
+            return o.isoformat()
+
+        if isinstance(o, Money):
+            return o.amount_as_string()
+
+        return super().default(o)
```

Comment:
What do you think to inherit from `django.core.serializers.json.DjangoJSONEncoder` then only manage `Money` instance ?